documentation/asciidoc/computers/compute-module/cm-emmc-flashing.adoc

```diff
@@ -8,17 +8,19 @@ IMPORTANT: For mass provisioning of CM3, CM3+ and CM4 the https://github.com/ras
 
 === Steps to Flash the eMMC 
 
-To flash the Compute Module eMMC, you either need a Linux system (a Raspberry Pi is recommended, or Ubuntu on a PC) or a Windows system (Windows 10 is recommended). For BCM2837 (CM3), a bug which affected the Mac has been fixed, so this will also work.
+To flash the Compute Module eMMC, you either need a Linux system (a Raspberry Pi is recommended, or Ubuntu on a PC) or a Windows system (Windows 10 is recommended).
 
-NOTE: There is a bug in the BCM2835 (CM1) bootloader which returns a slightly incorrect USB packet to the host. Most USB hosts seem to ignore this benign bug and work fine; we do, however, see some USB ports that don't work due to this bug. We don't quite understand why some ports fail, as it doesn't seem to be correlated with whether they are USB2 or USB3 (we have seen both types working), but it's likely to be specific to the host controller and driver. This bug has been fixed in BCM2837.
+NOTE: Due to a bug in Compute Module 1, it is not possible to flash its eMMC from a Mac.
+
+NOTE: There is a bug in the Compute Module 1 bootloader which returns a slightly incorrect USB packet to the host. A small number of USB host controllers object to this and cannot be used to flash a Compute Module 1.
 
 === Setting up the CMIO board
 
 ==== Compute Module 4
 
-Ensure the Compute Module is fitted correctly installed on the IO board. It should lie flat on the IO board.
+Ensure the Compute Module is fitted correctly on the IO board: It should lie flat on the IO board.
```

Comment:
First word after a colon should start with a lower-case letter. And IMHO a semi-colon here would be more appropriate than a colon.